Celeste.Mod.mm/Celeste.Mod.mm.csproj

```diff
@@ -51,6 +51,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="MAB.DotIgnore" Version="3.0.2" />
```

Comment:
Not sure there's any consistent convention for order of dependencies, but right at the top doesn't seem appropriate for this one.

Reply:
I just let Visual Studio do its thing. I'll move it to the bottom though